CorsixTH/Src/th_gfx.h

```diff
@@ -510,38 +510,43 @@ class animation_manager {
   void fix_next_frame(uint32_t iFirst, size_t iLength);
 };
 
+// Integer 2D pair.
+struct xy_pair {
```

Comment:
I have a future vision for different structs for different coordinate systems so they can't be mixed up. This is fine now, only used for pixel offsets current.